src/content/contribute/writing-a-scaffold.md

```diff
@@ -8,9 +8,20 @@ contributors:
   - rishabh3112
 ---
 
-Welcome to the demonstration of the new `webpack init` command! To view what we are building today, run `webpack init webpack-scaffold-demo`. This demo will show you how to build your own webpack scaffold. Let's start by creating a file named `generator.js`.
+A scaffold is a node module which exports a `yeoman generator` based upon [Scaffolding API](/guides/scaffolding/#api). Scaffolds are used to initialise a new webpack project, tailored to a specific use case. To view what we are building today, run `webpack init webpack-scaffold-demo`. This demo will show you how to build your own webpack scaffold. Let's start by creating a file named `generator.js`.
```

Comment:
node module is not correct, also scaffold could do more than that, so maybe rephrase a bit

```suggestion
A scaffold is an ECMAScript Module which exports a `yeoman generator` based on [Scaffolding API](/guides/scaffolding/#api). Scaffolds can be used but not limited to initialize a new webpack project, tailored to a specific use case. To view what we are building today, run `webpack init webpack-scaffold-demo`. This demo will show you how to build your own webpack scaffold. Let's start by creating a file named `generator.js`.
```